examples/one-2-one/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "http://json.schemastore.org/tsconfig",
+  "extends": "@loopback/build/config/tsconfig.common.json",
+  "include": [
+    "src",
+    "test",
+    "index.ts"
```

Comment:
We have recently reworked our project layout so that test files are kept in `src/__tests__`. Please rebase your pull request on top of the latest master and update your new example app to follow the new convention. See the following pull request for the changes you need to make: https://github.com/strongloop/loopback-next/pull/2316/files.

It basically boils down to:
 - move `test` to `src/__tests__`
 - update `package.json` scripts & files
 - update `tsconfig.build.json`
 - fix paths in `import` statements

I apologize for inconvenience I caused here.